validator/sawtooth_validator/consensus/proxy.py

```diff
@@ -34,9 +34,11 @@ class UnknownBlock(Exception):
 
 
 class ConsensusActivationObserver(ChainObserver):
-    """Observes chain updates and activates new engines, if they have been activated."""
+    """Observes chain updates and activates new engines, if they have been
+    activated."""
 
-    def __init__(self, consensus_registry, consensus_notifier, settings_view_factory):
+    def __init__(self, consensus_registry, consensus_notifier,
+                 settings_view_factory):
```

Comment:
The formatting changes in the "Notify ChainObservers of genesis block" commit should be squashed into the previous commit.

Reply:
fixed